source-linkedin-pages/acmeCo/flow.yaml

```diff
@@ -0,0 +1,34 @@
+---
+collections:
+  acmeCo/follower_statistics:
+    schema: follower_statistics.schema.yaml
+    key:
+      - /_meta/row_id
+  acmeCo/organization_lookup:
+    schema: organization_lookup.schema.yaml
+    key:
+      - /_meta/row_id
+  acmeCo/share_statistics:
+    schema: share_statistics.schema.yaml
+    key:
+      - /_meta/row_id
+  acmeCo/total_follower_count:
+    schema:
+      $schema: "http://json-schema.org/draft-04/schema#"
+      additionalProperties: true
+      type: object
+      properties:
+        firstDegreeSize:
+          type:
+            - "null"
+            - integer
+        _meta:
```

Comment:
We need `_meta` to be required on these schemas, so that the key `/_meta/row_id` is required